tests/poller_scaling_test.go

```diff
@@ -0,0 +1,237 @@
+		Spec: &nexuspb.EndpointSpec{
+			Name: endpointName,
+			Target: &nexuspb.EndpointTarget{
+				Variant: &nexuspb.EndpointTarget_Worker_{
+					Worker: &nexuspb.EndpointTarget_Worker{
+						Namespace: s.Namespace().String(),
+						TaskQueue: tq,
+					},
+				},
+			},
+		},
+	})
+	s.NoError(err)
+
+	// Queue up a couple workflows
+	for i := 0; i < 5; i++ {
+		_, err := s.sdkClient.ExecuteWorkflow(
+			ctx, sdkclient.StartWorkflowOptions{TaskQueue: tq}, "wf")
+		s.NoError(err)
+	}
+
+	// Poll for a task and see attached decision is to scale up b/c of backlog
+	feClient := s.FrontendClient()
+	// This needs to be done in an eventually loop because nexus endpoints don't become available immediately...
+	s.EventuallyWithT(func(t *assert.CollectT) {
+		resp, err := feClient.PollWorkflowTaskQueue(ctx, &workflowservice.PollWorkflowTaskQueueRequest{
+			Namespace: s.Namespace().String(),
+			TaskQueue: &taskqueuepb.TaskQueue{Name: tq, Kind: enumspb.TASK_QUEUE_KIND_NORMAL},
+		})
+		assert.NoError(t, err)
+		if assert.NotNil(t, resp.PollerScalingDecision) {
+			assert.GreaterOrEqual(t, int32(1), resp.PollerScalingDecision.PollRequestDeltaSuggestion)
+		}
+
+		// Start enough activities / nexus tasks to ensure we will see scale up decisions
+		commands := make([]*commandpb.Command, 0, 5)
+		for i := 0; i < 5; i++ {
+			commands = append(commands, &commandpb.Command{
+				CommandType: enumspb.COMMAND_TYPE_SCHEDULE_ACTIVITY_TASK,
+				Attributes: &commandpb.Command_ScheduleActivityTaskCommandAttributes{ScheduleActivityTaskCommandAttributes: &commandpb.ScheduleActivityTaskCommandAttributes{
+					ActivityId:          fmt.Sprintf("%v", i),
+					ActivityType:        &commonpb.ActivityType{Name: "test-activity-type"},
+					TaskQueue:           &taskqueuepb.TaskQueue{Name: tq, Kind: enumspb.TASK_QUEUE_KIND_NORMAL},
+					Input:               payloads.EncodeString("test-input"),
+					StartToCloseTimeout: durationpb.New(10 * time.Second),
+				}},
+			})
+			commands = append(commands, &commandpb.Command{
+				CommandType: enumspb.COMMAND_TYPE_SCHEDULE_NEXUS_OPERATION,
+				Attributes: &commandpb.Command_ScheduleNexusOperationCommandAttributes{
+					ScheduleNexusOperationCommandAttributes: &commandpb.ScheduleNexusOperationCommandAttributes{
+						Endpoint:  endpointName,
+						Service:   "service",
+						Operation: "operation",
+						Input:     s.mustToPayload("input"),
+					},
+				},
+			},
+			)
+		}
+		_, err = feClient.RespondWorkflowTaskCompleted(ctx, &workflowservice.RespondWorkflowTaskCompletedRequest{
+			Identity:  "test",
+			TaskToken: resp.TaskToken,
+			Commands:  commands,
+		})
+		assert.NoError(t, err)
+	}, 20*time.Second, 200*time.Millisecond)
+
+	// Wait a little time to ensure add rate exceeds dispatch rate & backlog age grows
+	time.Sleep(500 * time.Millisecond) //nolint:forbidigo // can't poll in an eventually, then we'd drain the tasks
```

Comment:
you can use Eventually with describetaskqueue to check backlog age:
https://github.com/dnr/temporal/blob/pri-light/tests/priority_fairness_test.go#L131-L144
does that work here?